<launch>
    <!--  Arguments for simulation launch -->
    <arg name="num_robots" default="1" /> <!-- 12, 24, 36 -->
    <arg name="robot_prerception_distance" default="0.2" /> 
    <arg name="robot_laser_ratio" default="0.1" /> 
    <arg name="map" default="real" />  <!-- map used on simulation-->
    <arg name="experiment_id" default="0" /> 
    <arg name="stoptime" default="300" /> 

    <param name="num_robots" type="int" value="$(arg num_robots)" />
    <param name="stoptime" type="int" value="$(arg stoptime)" />
    <param name="experiment_id" type="int" value="$(arg experiment_id)" />
    <param name="robot_prerception_distance" type="double" value="$(arg robot_prerception_distance)" />
    <param name="map_name" type="str" value="$(arg map)" />


    <!-- start visual capture -->
    <!-- <include file="$(find cimap)/epuck_driver/launch/usb_cam.launch">
    </include>

    <include file="$(find cimap)/epuck_driver/launch/ar_tracker.launch">
    </include> -->




    <!-- CIMAP NODE -->
    <node pkg="cimap" type="real_explorer.py" name='cimap_explorer' output="screen" required="true">

    </node>

</launch>